include/rocksdb.h

```diff
@@ -149,14 +161,38 @@ struct rocksdb_column_family_options_s {
 
   /** @since 2 */
   rocksdb_filter_policy_t filter_policy;
+
+  /** @since 3 */
+  bool optimize_filters_for_hits;
+
+  /** @since 3 */
+  int num_levels;
+
+  /** @since 3 */
+  int max_write_buffer_number;
+
+  /** @since 3 */
+  bool paranoid_file_checks;
+
+  /** @since 3 */
+  bool force_consistency_checks;
 };
 
-/** @version 0 */
+/** @version 1 */
 struct rocksdb_read_options_s {
   int version;
 
   /** @since 0 */
   rocksdb_snapshot_t *snapshot;
+
+  /** @since 1 */
+  bool async_io;
+
+  /** @since 1 */
+  bool verify_checksums;
```

Comment:
The same concerns as with `paranoid_checks` and `force_consistency_checks` apply here as well.